unicoremc/managers/settings.py

```diff
@@ -72,6 +72,13 @@ def write_frontend_settings(
         for lang in available_languages:
             languages.append(repr((lang.get_code(), lang.get_display_name())))
 
+        if hub_app:
+            hub_app_id = hub_app.get('uuid')
```

Comment:
or perhaps this instead of the _if block_ `hub_app_id = hub_app.get('uuid') if hub_app else None`